downstream/modules/platform/ref-controller-variables.adoc

```diff
@@ -6,7 +6,7 @@
 |====
 | *Variable* | *Description* 
 | *`admin_password`* | The password for an administration user to access the UI upon install completion.
-| *`automationcontroller_main_url`* | For an alternative front end URL needed for SSO configuration with {CatalogName}, provide the URL.
+| *`automation_controller_main_url`* | For an alternative front end URL needed for SSO configuration with {CatalogName}, provide the URL.
```

Comment:
If this is for 2.4 references to {CatalogNameStart} should be removed.

Reply:
Hi Ian, thank you for your review--I'll remove {CatalogName} in a separate PR. That way I can check for any other occurrences in 2.4. Would you be willing to squash and merge this change in the meantime?